jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/RemoveDataCaseFileInstanceCommand.java

```diff
@@ -40,9 +42,11 @@
 
     private List<String> variableNames;
     private AuthorizationManager authorizationManager;
+    private Long processInstanceId;
     
-    public RemoveDataCaseFileInstanceCommand(IdentityProvider identityProvider, List<String> variableNames, AuthorizationManager authorizationManager) {                
+    public RemoveDataCaseFileInstanceCommand(Long processInstanceId, IdentityProvider identityProvider, List<String> variableNames, AuthorizationManager authorizationManager) {                
```

Comment:
this will required a full downstream compilation at least. Commands are sensitive. @gmunozfe

Reply:
this is the only way I found to access variable readonly definition in RemoveDataCaseFileInstace. Lets do the full downstream compilation